core/com.b2international.snowowl.core.rest/.classpath

```diff
@@ -1,29 +1,24 @@
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src/"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/commons-fileupload-1.4.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/commons-logging-1.2.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-aop-5.3.19.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-beans-5.3.19.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-context-5.3.19.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-core-5.3.19.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/springdoc-openapi-common-1.5.9.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/springdoc-openapi-webmvc-core-1.5.9.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-expression-5.3.19.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-security-config-5.6.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-security-core-5.6.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-security-web-5.6.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-security-crypto-5.6.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-web-5.3.19.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-webmvc-5.3.19.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/swagger-core-2.1.9.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-boot-autoconfigure-2.5.2.jar"/>
-	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-boot-2.5.2.jar"/>
-	<classpathentry kind="output" path="target/classes/"/>
+	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-beans-6.1.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-context-6.1.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-core-6.1.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-expression-6.1.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-security-config-6.2.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-aop-6.1.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-security-core-6.2.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="WEB-INF/lib/spring-security-web-6.2.1.jar" sourcepath="C:/Users/mark/.m2/repository/org/springframework/security/spring-security-web/6.2.1/spring-security-web-6.2.1-sources.jar"/>
```

Comment:
⚠️ Source attachment on classpath entry points to local path

Reply:
😭